resources/home/dnanexus/run_workflows/utils/manage_dict.py

```
@@ -1051,3 +1077,25 @@ def populate_tso500_reports_workflow(
+    Returns
+    -------
+    bool
+        True if any 'hold' key is set to True or "true", else False
+    """
+    # Find all keys named 'hold' in the nested config
+    found = search_exact_key('hold', config)
+    # Check if any of those keys are set to True or "true" in found
+    for value in found:
+        if value is True:
+            return True
+    return False
```

Ruff (0.12.2)

Use `return any(value is True for value in found)` instead of `for` loop

Replace with `return any(value is True for value in found)`

(SIM110)